docs/src/tutorials/Installation_and_configuration.md

```diff
@@ -0,0 +1,308 @@
+# Getting started with thin-edge.io on a Raspberry Pi
```

Comment:
Yet another change that would be good to have: rename the file to match the actual content. `getting-started.md`

Reply:
I added an extr a line to the TOC: getting started guide